design-patterns/gc-institution/gc-institution-en.html

```diff
@@ -0,0 +1,90 @@
+---
+title: About the institution pattern
+description: Documentation and working example for the about the institution pattern
+language: en
+altLangPage: gc-institution-fr.html
+dateModified: 2023-08-09
+layout: no-container
+---
+<div class="container">
+	<h1 property="name" id="wb-cont" dir="ltr">{{ page.title }}</h1>
+
+	<span class="wb-prettify all-pre hide"></span>
+	<p>TThe about the institution pattern is a section on an institutional landing page that provides links to corporate, program and policy information.</p>
+
+	<h2>On this page</h2>
+	<ul>
+		<!-- <li><a href="#inc-components">Included components</a></li> -->
+		<li><a href="#before-start">Before you start</a></li>
+		<li><a href="#accessibility-guidance">Accessibility guidance</a></li>
+		<li><a href="#variations">Variations</a></li>
+		<li><a href="#eval-report">Evaluation and report</a></li>
+		<li><a href="#technical-docs">Technical documentation</a></li>
+	</ul>
+
+	<h2 id="before-start">Before you start</h2>
+	<p>About the institution is a mandatory pattern on an institutional landing page. It must include mandate and transparency links.</p>
+
+	<ul>
+		<li><a href="">How to set up the institutional landing page</a></li>
```

Comment:
Is this link intended to redirect to the design guidance?

Reply:
Ok, the link will need to be updated here once it is ready. I will keep this change request open until it is.

Reply:
Added the link, it's the old content, but it's going to the right place when we update everything together